etc/config/fortran.amazon.properties

```diff
@@ -244,22 +244,23 @@ compiler.flangtrunknew.supportsBinary=false
 # GCC for ARM 64bit
 group.gccaarch64.compilers=farm64g640:farm64g730:farm64g820:farm64g1210:farm64g1220
 group.gccaarch64.groupName=ARM (AARCH64) GCC
+group.gccaarch64.baseName=AARCH64 gfortran
 compiler.farm64g640.exe=/opt/compiler-explorer/arm64/gcc-6.4.0/aarch64-unknown-linux-gnu/bin/aarch64-unknown-linux-gnu-gfortran
-compiler.farm64g640.name=AARCH64 gfortran 6.4
+compiler.farm64g640.semver=6.4
 compiler.farm64g730.exe=/opt/compiler-explorer/arm64/gcc-7.3.0/aarch64-unknown-linux-gnu/bin/aarch64-unknown-linux-gnu-gfortran
-compiler.farm64g730.name=AARCH64 gfortran 7.3
+compiler.farm64g730.semver=7.3
 compiler.farm64g820.exe=/opt/compiler-explorer/arm64/gcc-8.2.0/aarch64-unknown-linux-gnu/bin/aarch64-unknown-linux-gnu-gfortran
-compiler.farm64g820.name=AARCH64 gfortran 8.2
+compiler.farm64g820.semver=8.2
 
 compiler.farm64g1210.exe=/opt/compiler-explorer/arm64/gcc-12.1.0/aarch64-unknown-linux-gnu/bin/aarch64-unknown-linux-gnu-gfortran
-compiler.farm64g1210.name=AARCH64 gfortran 12.1.0
-compiler.farm64g1210.exe=/opt/compiler-explorer/arm64/gcc-12.1.0/aarch64-unknown-linux-gnu/bin/aarch64-unknown-linux-gnu-objdump
+compiler.farm64g1210.objdumper=/opt/compiler-explorer/arm64/gcc-12.1.0/aarch64-unknown-linux-gnu/bin/aarch64-unknown-linux-gnu-objdump
+compiler.farm64g1210.demangler=/opt/compiler-explorer/arm64/gcc-12.1.0/aarch64-unknown-linux-gnu/bin/aarch64-unknown-linux-gnu-c++filt
```

Comment:
@partouf I saw that the compiler after this one defines its own demangler too so I added it here as well, not sure if needed or if it was left like this on pourpose

Reply:
mangling is usually the same, but it makes sense to make it consistent